pkgs/by-name/si/simg2img/package.nix

```diff
@@ -13,13 +19,42 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ zlib ];
 
-  makeFlags = [ "PREFIX=$(out)" "DEP_CXX:=$(CXX)" ];
+  makeFlags = [
+    "PREFIX=$(out)"
+    "DEP_CXX:=$(CXX)"
+  ];
+
+  patches = [
+    (fetchpatch {
+      name = "PR-38-Fix-for-Apple-Silicon.patch";
```

Comment:
This one is included in 1.1.5, maybe we should just update to that version?

Reply:
Nice!